### PR TITLE
Use singular number for all feature names

### PR DESCRIPTION
**Pull request by asomers**
features => feature
users => user

Neither of these features have yet been included in a release, so it's
ok to rename them.

### CODE REVIEW

**Review by rtzoeller — APPROVED** [review]
bors r+

**Comment by asomers** [comment]
Looks like rustc itself is broken for uclibc .  https://github.com/rust-lang/rust/commit/33fd73fedeb5cd61dbbb7562fd927c0b6994653a .

**Comment by rtzoeller** [comment]
@asomers thanks for filing https://github.com/rust-lang/rust/issues/95866.

FYI @skrap.

File: .cirrus.yml [hunk 1]
```diff
@@ -289,6 +289,9 @@ task:
     - name: Linux armv7 uclibceabihf
       env:
         TARGET: armv7-unknown-linux-uclibceabihf
+        # Pin nightly compiler due to
+        # https://github.com/rust-lang/rust/issues/95866
+        TOOLCHAIN: nightly-2022-04-01
```
Comment (rtzoeller) [inline .cirrus.yml:294]: I believe you also need the 

```
setup_script:
    - rustup target add $TARGET
    - rustup toolchain install $TOOLCHAIN --profile minimal --target $TARGET
    - rustup component add --toolchain $TOOLCHAIN clippy
```

bit.

**Comment by asomers** [comment]
Ok, I think I got CI straightened out.

**Comment by rtzoeller** [comment]
bors retry

**Pull request merged**